src/core/layertree/qgscolorramplegendnode.h

```diff
@@ -43,9 +43,12 @@ class CORE_EXPORT QgsColorRampLegendNode : public QgsLayerTreeModelLegendNode
      * \param minimumLabel label text to render for the minimum value in the ramp
      * \param maximumLabel label text to render for the maximum value in the ramp
      * \param parent attach a parent QObject to the legend node.
+     * \param key rule key
+     * \param parentKey parent rule key
```

Comment:
```suggestion
     * \param key rule key (since QGIS 3.36)
     * \param parentKey parent rule key (since QGIS 3.36)
```

Reply:
Also would be better to add some explanatory docs here, eg for when parent rule key should be set.